readme.txt
=== Shophistic Lite ===
Contributors: nicoandrade
Tags: e-commerce, blog, featured-images, two-columns, threaded-comments, custom-menu, left-sidebar, sticky-post, translation-ready, custom-background, full-width-template, rtl-language-support, theme-options
Requires at least: 4.9.6
Tested up to: WordPress 5.3.2
Stable tag: 1.0.25
License: GPLv2 or later
License URI: http://www.gnu.org/licenses/gpl-2.0.html

Shophistic Lite is the perfect theme for your shop online.


== Description ==
Shophistic Lite is the perfect theme for your shop online. With a beautiful and unique design, is responsive and fully compatible with WooCommerce. The easiest way to create your store online. If you need more features you should check the PRO version, Shophistic: https://www.quemalabs.com/theme/shophistic/


== Installation ==
1. In your admin panel, go to Appearance -> Themes and click the Add New button.
2. Click Upload and Choose File, then select the theme's ZIP file. Click Install Now.
3. Click Activate to use your new theme right away.


== Documentation ==
* Social Menu
Using the social nav menu is just like using any other nav menu in WordPress. You just need to add custom links pointing to your social media profiles.

So, the first step would be to create a new menu under the “Appearance > Menus” screen in the WordPress admin. To add custom links to this menu, use the “Links” box. Just make a link to the social media profiles you want to display.

And, for the last part, be sure to select “Social” for the “Theme Location” before saving your menu.

* Where do I find the Demo XML import file?
You can download it from here: http://cl.ly/3h2K0P373B11

* How to get the PRO version of Shophistic Lite
The PRO version is called "Shophistic" and includes tons of more features. You can buy it here:
http://www.quemalabs.com/theme/shophistic/


== Development ==
You can find Shophistic Lite in Github and create issues or improvements there:
https://github.com/nicoandrade/Shophistic-Lite


== Changelog ==

= 1.0.25 - 20/03/2020 =
* Fixed: Columns of products on the shop page

= 1.0.24 - 09/01/2020 =
* Updated: Go PRO button on customizer

= 1.0.23 - 01/07/2019 =
* Improved: Go PRO button on customizer

= 1.0.22 - 23/04/2019 =
* Added: WooCommerce 3.6.1 support

= 1.0.21 - 04/02/2019 =
* Improved: CSS class for custom variations colors
* Fixed: Header CSS issue on small desktop screens
* Changed: readme.txt
* Changed: changelog now is on readme.txt file

= 1.0.20 - 11/12/2018 =
* Fixed: Search products button
* Fixed: Slidemenu bug (@dontbyte)
* Added: WooCommerce 3.5.2 support
* Improved: Permalink for post image (@lenivene)

= 1.0.19 - 30/05/2018 =
* Fixed: Text domain

= 1.0.18 - 30/05/2018 =
* Added: WooCommerce 3.4 support

= 1.0.17 - 29/06/2017 =
* Added: WooCommerce 3.1 support

= 1.0.16 - 18/06/2017 =
* Fixed: Issues with TinyMCE CSS

= 1.0.15 - 02/06/2017 =
* Fixed: Images switch on variable products

= 1.0.14 - 15/04/2017 =
* Added: WooCommerce 3.0.3 support

= 1.0.13 - 12/04/2017 =
* Added: WooCommerce 3 support
* Fixed: Error on some server with woocommerce_support.php
* Improved: On changing variations the slider goes to the first slide
* Improved: Now the Cart button links to Cart page on mobile
* Updated: TGM Class

= 1.0.12 - 20/07/2016 =
* Updated: WooCommerce templates
* Fixed: Variable products with HTML characters on variations

= 1.0.11 - 15/07/2016 =
* Updated: WooCommerce template

= 1.0.10 - 14/04/2016 =
* Fixed: issue with variation but no taxonomies

= 1.0.9 - 21/03/2016 =
* Fixed: CSS issue on hovering the Cart

= 1.0.8 - 18/02/2016 =
* Improved: on mobile now you see the products at the top and sidebar at the bottom

= 1.0.7 - 21/01/2016 =
* Fixed: add to cart issue with WoCommerce 2.5
* Updated: WooCommerce templates

= 1.0.6 - 15/12/2015 =
* Fixed: issue with selecting variations on products (ex: colors or sizes)

= 1.0.5 - 26/11/2015 =
* Added: review button on customizer
* Improve: theme description

= 1.0.4 - 14/11/2015 =
* Fixed: minor error when WooCommerce is not active

= 1.0.3 - 13/11/2015 =
* Fixed: Translatable strings on footer

= 1.0.2 - 30/09/2015 =
* Fixed: CSS on WooCommerce products archive

= 1.0.1 - 30/09/2015 =
* Complete Theme Update

= 1.0.0 - 09/09/2015 =
* Released


== Resources ==
Font Awesome icon font, Created by Dave Gandy
Font Awesome licensed under SIL OFL 1.1 · Code licensed under MIT License
Source: http://fortawesome.github.io/

Bootstrap Created by @mdo and @fat.
Code licensed under MIT
Source: http://getbootstrap.com

Modernizr 2.8.3 (Custom Build) | MIT & BSD
Source: http://modernizr.com

Pace
License: https://raw.githubusercontent.com/HubSpot/pace/master/LICENSE
Source: http://github.hubspot.com/pace/docs/welcome/

HTML5 Shiv by @afarkas @jdalton @jon_neal @rem
License: MIT/GPL2 Licensed
Source: https://github.com/afarkas/html5shiv

PhotoSwipe by Dmitry Semenov
License: MIT license
Source: https://github.com/dimsemenov/photoswipe

hoverIntent
License: MIT license
Source: http://cherne.net/brian/resources/jquery.hoverIntent.html

Isotope & Packery by metafizzy
License: GPLv3
Source: isotope.metafizzy.co/

imagesLoaded by @Darsain and @desandro
License: MIT license
Source: https://github.com/desandro/imagesloaded

[Lato Font](https://www.google.com/fonts/specimen/Lato) by Łukasz Dziedzic - Licensed under the [SIL Open Font License, 1.1](http://scripts.sil.org/OFL).

Images used in the demo site are from Unsplash
License: https://unsplash.com/license
Source: https://unsplash.com

changelog.txt
Version: 4.10.0
  - Updated Bootstrap to version 5.2.3
  - Updated all outdated third party vendor libraries to their latest versions

Version: 4.9.1
  - Updated Bootstrap to version 5.2.2
  - Updated all outdated third party vendor libraries to their latest versions

Version: 4.9.0
  - Updated Bootstrap to version 5.2.1
  - Updated all outdated third party vendor libraries to their latest versions
  - Improved PHP Email From library to ignore server header responses like 409

Version: 4.8.0
  - Updated Bootstrap to version 5.2.0
  - Updated all outdated third party vendor libraries to their latest versions

Version: 4.7.1
  - Section typo fix (cliens -> clients)

Version: 4.7.0
  - Updated Bootstrap to version 5.1.3
  - Updated all outdated third party vendor libraries to their latest versions

Version: 4.6.0
  - Updated Bootstrap to version 5.1.2
  - Updated all outdated third party vendor libraries to their latest versions

Version: 4.5.0
  - Fixed slider issue in testimonials and portfolio details sections

Version: 4.4.0
  - Updated Bootstrap to version 5.1.1
  - Updated all outdated third party vendor libraries to their latest versions
  - Improved and updated dev version gulp scripts

Version: 4.3.0
  - Updated Bootstrap to version 5.0.1
  - Updated all outdated third party vendor libraries to their latest versions
  - Fixed navigation links focus color

Version: 4.2.0
  - Updated Bootstrap to version 5.0.0 Final
  - Updated all outdated third party vendor libraries to their latest versions

Version: 4.1.0
  - Updated Bootstrap to version 5.0.0-beta3
  - Updated all outdated third party vendor libraries to their latest versions
  - Updated the PHP Email Form to V3.1

Version: 4.0.1
  - Updated Bootstrap to version 5.0.0-beta2
  - Updated all outdated third party vendor libraries to their latest versions

Version: 4.0.0
  - The template does not require jQuery anymore
  - Removed jQuery and all the jQuery plugins
  - The assets/js/main.js was rewritten completely with vanilla Javascript. No more jQuery code
  - Restructured the dev version for better development experience
  - Updated the PHP Email Form to V3.0 - No jQuery dependency. Added attachment support

Version: 3.0.3
  - Updated Bootstrap to version 5.0.0-beta1
  - Updated the PHP Email Form to v2.3
  - Other small fixes and improvements
  
Version: 3.0.2
  - Updated Bootstrap to version 5.0.0-alpha3
  - Updated all outdated third party vendor libraries to their latest versions

Version: 3.0.1
  - Update Bootstrap v5.0 to Alpha 2
  - Updated all outdated third party vendor libraries to their latest versions

Version: 3.0.0
  - Initial release with using the Bootstrap v5.0 (Alpha)

Version: 2.2.0
  - Updated the PHP Email Form to v2.1
  - Other small fixes and improvements

Version: 2.1.0
  - Updated Bootstrap to version 4.5.0
  - Updated the PHP Email Form library to version 2.0 with reCaptcha support
  - Aded inner-page.html tempalte
  - Updated all outdated third party vendor libraries to their latest versions
  - Other small fixes and improvements
  
Version: 2.0.0
  - The template was rebuilt from scratch with the latest Bootstrap version (4.4.1)
  - Added SMPTP support for the contact form script (Pro)
  - Added NodeJS NPM Development version (Pro unlimited & Membership members)
  
Version: 1.0.0
  - Initial Release


mail.php
<?php

    // Only process POST reqeusts.
    if ($_SERVER["REQUEST_METHOD"] == "POST") {
        // Get the form fields and remove whitespace.
        $name = strip_tags(trim($_POST["name"]));
				$name = str_replace(array("\r","\n"),array(" "," "),$name);
        $email = filter_var(trim($_POST["email"]), FILTER_SANITIZE_EMAIL);
        $subject = trim($_POST["subject"]);
        $message = trim($_POST["message"]);

        // Check that data was sent to the mailer.
        if ( empty($name) OR empty($subject) OR empty($message) OR !filter_var($email, FILTER_VALIDATE_EMAIL)) {
            // Set a 400 (bad request) response code and exit.
            http_response_code(400);
            echo "Please complete the form and try again.";
            exit;
        }

        // Set the recipient email address.
        // FIXME: Update this to your desired email address.
        $recipient = "name@domain.com";

        // Set the email subject.
        $subject = "$subject";

        // Build the email content.
        $email_content = "Name: $name\n";
        $email_content .= "Email: $email\n\n";
        $email_content .= "Subject: $subject\n\n";
        $email_content .= "Message:\n$message\n";

        // Build the email headers.
        $email_headers = "From: $name <$email>";
       
        // mysql.

        $conn = mysql_connect("localhost","root","");
    mysql_select_db("contact_form",$conn);
    mysql_query("INSERT INTO tbl_contact (fullname, email, user_message) VALUES ('" . $fullname. "', '" . $email. "','" . $user_message. "')");
    $insert_id = mysql_insert_id();
        // connecting to mysql end.

        // Send the email.
        if (mail($recipient, $subject, $email_content, $email_headers)) {
            // Set a 200 (okay) response code.
            http_response_code(200);
            echo "Thank You! Your message has been sent.";
        } else {
            // Set a 500 (internal server error) response code.
            http_response_code(500);
            echo "Oops! Something went wrong and we couldn't send your message.";
        }

    } else {
        // Not a POST request, set a 403 (forbidden) response code.
        http_response_code(403);
        echo "There was a problem with your submission, please try again.";
    }

?>
<?php
if(!empty($_POST["submit"])) {
    $fullname = $_POST["fullname"];
    $email = $_POST["email"];
    $user_message = $_POST["user_message"];

    $conn = mysql_connect("localhost","root","");
    mysql_select_db("contact_form",$conn);
    mysql_query("INSERT INTO tbl_contact (fullname, email, user_message) VALUES ('" . $fullname. "', '" . $email. "','" . $user_message. "')");
    $insert_id = mysql_insert_id();
    if(!empty($insert_id)) {
    $message = "Successfully Added.";
    }
}
?>

header.php
<!DOCTYPE html>
<html <?php language_attributes(); ?> class="no-js">

<head>
    <meta charset="<?php bloginfo( 'charset' ); ?>">
    <link rel="profile" href="http://gmpg.org/xfn/11">
    <link rel="pingback" href="<?php bloginfo( 'pingback_url' ); ?>">

    <!--[if lt IE 9]>
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <script src="<?php echo esc_url( get_template_directory_uri() ); ?>/js/html5shiv.js"></script>
    <![endif]-->

    <meta name="description" content="<?php bloginfo( 'description' ); ?>">

    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <!-- WP_Head -->
    <?php wp_head(); ?>
    <!-- /WP_Head -->
</head>



<body <?php body_class(); ?>>

    <div id="wrap">

        <header id="header">
            <div class="container">
                <div class="row">

                    <div class="ql_logo_nav col-md-9 col-xs-12">

                        <div class="logo_container">
                            <a href="<?php echo home_url(); ?>/" class="ql_logo google-font">
                                <?php bloginfo( 'name' ); ?>
                            </a>
                            <button id="ql_nav_btn" type="button" class="navbar-toggle collapsed" data-toggle="collapse"
                                data-target="#ql-navigation" aria-expanded="false">
                                <span class="sr-only">Toggle navigation</span>
                                <span class="icon-bar"></span>
                                <span class="icon-bar"></span>
                                <span class="icon-bar"></span>
                            </button>
                        </div><!-- /logo_container -->

                        <div class="collapse navbar-collapse" id="ql-navigation">
                            <nav id="jqueryslidemenu" class="jqueryslidemenu navbar " role="navigation">
                                <?php
                                        if ( has_nav_menu( 'menu-1' ) ) {
                                            wp_nav_menu( array(
                                                'theme_location' => 'menu-1',
                                                'depth'          => 3,
                                                'menu_class'     => 'nav navbar-nav',
                                                'fallback_cb'    => 'wp_bootstrap_navwalker::fallback',
                                                'walker'         => new wp_bootstrap_navwalker(),
                                            ) );
                                        } else {
                                            echo "<ul id='nav' class='nav navbar-nav'>";
                                            wp_list_pages( array(
                                                'title_li' => '',
                                            )
                                            );
                                            echo "</ul>";
                                        }
                                        ;
                                    ?>
                            </nav>
                        </div>
                        <div class="clearfix"></div>

                    </div><!-- col-md-8 -->

                    <?php
                            if ( in_array( 'woocommerce/woocommerce.php', apply_filters( 'active_plugins', get_option( 'active_plugins' ) ) ) ) {
                            ?>
                    <div class="login_cart_wrap col-md-3 col-xs-12">

                        <div class="ql_cart_wrap">
                            <button href="<?php echo esc_url( WC()->cart->get_cart_url() ); ?>" class="ql_cart-btn">
                                <?php echo wp_kses_post( WC()->cart->get_cart_total() ); ?>
                                <span class="count">(<?php echo esc_html( WC()->cart->cart_contents_count ); ?>)</span>
                                <i class="ql-bag"></i><i class="ql-chevron-down"></i>
                            </button>

                            <div id="ql_woo_cart">
                                <?php global $woocommerce; ?>

                                <?php the_widget( 'WC_Widget_Cart' ); ?>
                            </div><!-- /ql_woo_cart -->
                        </div>
                        <div class="login_btn_wrap">
                            <?php if ( is_user_logged_in() ) { ?>
                            <a class="ql_login-btn"
                                href="<?php echo esc_url( get_permalink( get_option( 'woocommerce_myaccount_page_id' ) ) ); ?>"
                                title="<?php esc_attr_e( 'My Account', 'shophistic-lite' ); ?>"><?php esc_html_e( 'My Account', 'shophistic-lite' ); ?></a>
                            <?php } else { ?>
                            <a class="ql_login-btn"
                                href="<?php echo esc_url( get_permalink( get_option( 'woocommerce_myaccount_page_id' ) ) ); ?>"
                                title="<?php esc_attr_e( 'Login', 'shophistic-lite' ); ?>"><?php esc_html_e( 'Login', 'shophistic-lite' ); ?></a>
                            <?php } ?>
                        </div>
                        <div class="clearfix"></div>
                    </div><!-- col-md-4 -->
                    <?php } //if WooCommerce active ?>

                </div><!-- row-->
            </div><!-- /container -->

        </header>

        <div class="clearfix"></div>

assets1/js/main.js
(function ($) {
    "use strict";

    /*:::::::::::::::::::::::::::::::::::
            Navbar Area
    :::::::::::::::::::::::::::::::::::*/

     // Navbar Sticky
    $(window).scroll(function () {
        var scroll = $(window).scrollTop();

        if (scroll >= 1) {
            $(".navbar").addClass("bg-primari");
        } else {
            $(".navbar").removeClass("bg-primari");
        }
    });


    //Smoth Scroll
    $(function () {
        $('.nav-link, .smoth-scroll').on('click', function (event) {
            var $anchor = $(this);
            $('html, body').stop().animate({
                scrollTop: $($anchor.attr('href')).offset().top - 0
            }, 1000);
            event.preventDefault();
        });
    });

    /*==========================
        Hero Area Slider
    ============================*/

    $('.hero-area-slids').owlCarousel({
        items: 1,
        loop: true,
        nav: false,
        doots: false,
        autoplay: true,
        animateOut: 'fadeOutRight',
        animateIn: 'fadeIn'

    });
    //Wow Animation
    new WOW().init();
    /*==========================
        Hero Title typer
    ============================*/
    var element = $('.typed');

    $(function () {
        element.typed({
            strings: ["Web Designer.", "Graphic Designer."],
            typeSpeed: 100,
            loop: true,
            autoplay: true,
        });
    });

    /*::::::::::::::::::::::::::::::::::::
       Portfolio Section
    ::::::::::::::::::::::::::::::::::::*/

    lightbox.option({
        'imageFadeDuration': 800,
        'resizeDuration': 500,
        'wrapAround': true
    });

    $('.portfolio-area').mixItUp();


    /*::::::::::::::::::::::::::::::::::::
       Testimonial Section
    ::::::::::::::::::::::::::::::::::::*/

    $('.testimonials').owlCarousel({
        items: 1,
        loop: true,
        autoplay: true,
        nav: true,
        navText: ['<i class="fa fa-angle-left"></i>', '<i class="fa fa-angle-right"></i>'],
        dots: false
    });


    /*::::::::::::::::::::::::::::::::::::
       Contact Area 
    ::::::::::::::::::::::::::::::::::::*/
    var form = $('#contact-form');

    var formMessages = $('.form-message');
    $(form).submit(function (e) {
        e.preventDefault();
        var formData = $(form).serialize();
        $.ajax({
                type: 'POST',
                url: $(form).attr('action'),
                data: formData
            })
            .done(function (response) {
                $(formMessages).removeClass('error');
                $(formMessages).addClass('success');
                $(formMessages).text(response);

                $('#contact-form input,#contact-form textarea').val('');
            })
            .fail(function (data) {
                $(formMessages).removeClass('success');
                $(formMessages).addClass('error');

                if (data.responseText !== '') {
                    $(formMessages).text(data.responseText);
                } else {
                    $(formMessages).text('Oops! An error occured and your message could not be sent.');
                }
            });
    });
    
    
    /*::::::::::::::::::::::::::::::::::::
    Preloader
    ::::::::::::::::::::::::::::::::::::*/
    $(window).on('load', function () {
        $('.preloader').fadeOut();
    });

}(jQuery));

/*::::::::::::::::::::::::::::::::::::
    Project request form
    ::::::::::::::::::::::::::::::::::::*/

 new JotformFeedback({ type: 2, width: 700, height:900, fontColor: "#FFFFFF", background: "#F59202", isCardForm: false, formId: "230088999574475" , buttonText: "Project request form", buttonSide: "bottom", buttonAlign: "right", base: "https://form.jotform.com/", }); 
 var ifr = document.getElementById("lightbox-230088999574475");
 if (ifr) { 
       var src = ifr.src; var iframeParams = []; if (window.location.href && window.location.href.indexOf("?") > -1) { iframeParams = iframeParams.concat(window.location.href.substr(window.location.href.indexOf("?") + 1).split('&')); } if (src && src.indexOf("?") > -1) { iframeParams = iframeParams.concat(src.substr(src.indexOf("?") + 1).split("&")); src = src.substr(0, src.indexOf("?")) } iframeParams.push("isIframeEmbed=1"); ifr.src = src + "?" + iframeParams.join('&'); } window.handleIFrameMessage = function(e) { if (typeof e.data === 'object') { return; } var args = e.data.split(":"); if (args.length > 2) { iframe = document.getElementById("lightbox-" + args[(args.length - 1)]); } else { iframe = document.getElementById("lightbox"); } if (!iframe) { return; } switch (args[0]) { case "scrollIntoView": iframe.scrollIntoView(); break; case "setHeight": iframe.style.height = args[1] + "px"; if (!isNaN(args[1]) && parseInt(iframe.style.minHeight) > parseInt(args[1])) { iframe.style.minHeight = args[1] + "px"; } break; case "collapseErrorPage": if (iframe.clientHeight > window.innerHeight) { iframe.style.height = window.innerHeight + "px"; } break; case "reloadPage": window.location.reload(); break; case "loadScript": if( !window.isPermitted(e.origin, ['jotform.com', 'jotform.pro']) ) { break; } var src = args[1]; if (args.length > 3) { src = args[1] + ':' + args[2]; } var script = document.createElement('script'); script.src = src; script.type = 'text/javascript'; document.body.appendChild(script); break; case "exitFullscreen": if (window.document.exitFullscreen) window.document.exitFullscreen(); else if (window.document.mozCancelFullScreen) window.document.mozCancelFullScreen(); else if (window.document.mozCancelFullscreen) window.document.mozCancelFullScreen(); else if (window.document.webkitExitFullscreen) window.document.webkitExitFullscreen(); else if (window.document.msExitFullscreen) window.document.msExitFullscreen(); break; } var isJotForm = (e.origin.indexOf("jotform") > -1) ? true : false; if(isJotForm && "contentWindow" in iframe && "postMessage" in iframe.contentWindow) { var urls = {"docurl":encodeURIComponent(document.URL),"referrer":encodeURIComponent(document.referrer)}; iframe.contentWindow.postMessage(JSON.stringify({"type":"urls","value":urls}), "*"); } }; window.isPermitted = function(originUrl, whitelisted_domains) { var url = document.createElement('a'); url.href = originUrl; var hostname = url.hostname; var result = false; if( typeof hostname !== 'undefined' ) { whitelisted_domains.forEach(function(element) { if( hostname.slice((-1 * element.length - 1)) === '.'.concat(element) || hostname === element ) { result = true; } }); return result; } };
 if (window.addEventListener) {
 window.addEventListener("message", handleIFrameMessage, false);
 }
 else if (window.attachEvent) { window.attachEvent("onmessage", handleIFrameMessage); } 






Contact_me/Contact_me -.js

var favicon = document.querySelector('link[rel="shortcut icon"]');
window.isDarkMode = (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches);
  if(favicon && window.isDarkMode) {
      favicon.href = favicon.href.replaceAll('favicon-2021-light%402x.png', 'favicon-2021-dark%402x.png');

  }

  JotForm.newDefaultTheme = false;
  JotForm.extendsNewTheme = false;
  JotForm.singleProduct = false;
  JotForm.newPaymentUIForNewCreatedForms = false;
  JotForm.highlightInputs = false;
  JotForm.clearFieldOnHide="disable";

  JotForm.init(function(){
  /*INIT-START*/
if (window.JotForm && JotForm.accessible) $('input_4').setAttribute('tabindex',0);
      setTimeout(function() {
          $('input_4').hint('Full Name');
       }, 20);
      setTimeout(function() {
          $('input_5').hint('Email Address');
       }, 20);
if (window.JotForm && JotForm.accessible) $('input_14').setAttribute('tabindex',0);
if (window.JotForm && JotForm.accessible) $('input_10').setAttribute('tabindex',0);
      JotForm.alterTexts({"ageVerificationError":"You must be older than {minAge} years old to submit this form.","alphabetic":"This field can only contain letters","alphanumeric":"This field can only contain letters and numbers.","appointmentSelected":"You’ve selected {time} on {date}","ccDonationMinLimitError":"Minimum amount is {minAmount} {currency}","ccInvalidCVC":"CVC number is invalid.","ccInvalidExpireDate":"Expire date is invalid.","ccInvalidExpireMonth":"Expiration month is invalid.","ccInvalidExpireYear":"Expiration year is invalid.","ccInvalidNumber":"Credit Card Number is invalid.","ccMissingDetails":"Please fill up the credit card details.","ccMissingDonation":"Please enter numeric values for donation amount.","ccMissingProduct":"Please select at least one product.","characterLimitError":"Too many Characters.  The limit is","characterMinLimitError":"Too few characters. The minimum is","confirmClearForm":"Are you sure you want to clear the form","confirmEmail":"E-mail does not match","currency":"This field can only contain currency values.","cyrillic":"This field can only contain cyrillic characters","dateInvalid":"This date is not valid. The date format is {format}","dateInvalidSeparate":"This date is not valid. Enter a valid {element}.","dateLimited":"This date is unavailable.","disallowDecimals":"Please enter a whole number.","doneButton":"Done","doneMessage":"Well done! All errors are fixed.","dragAndDropFilesHere_infoMessage":"Drag and drop files here","email":"Enter a valid e-mail address","error":"Error","fieldError":"field has an error.","fillMask":"Field value must fill mask.","freeEmailError":"Free email accounts are not allowed","generalError":"There are errors on the form. Please fix them before continuing.","generalPageError":"There are errors on this page. Please fix them before continuing.","geoNotAvailableDesc":"Location provider not available. Please enter the address manually.","geoNotAvailableTitle":"Position Unavailable","geoPermissionDesc":"Check your browser's privacy settings.","geoPermissionTitle":"Permission Denied","geoTimeoutDesc":"Please check your internet connection and try again.","geoTimeoutTitle":"Timeout","gradingScoreError":"Score total should only be less than or equal to","incompleteFields":"There are incomplete required fields. Please complete them.","inputCarretErrorA":"Input should not be less than the minimum value:","inputCarretErrorB":"Input should not be greater than the maximum value:","justSoldOut":"Just Sold Out","lessThan":"Your score should be less than or equal to","maxDigitsError":"The maximum digits allowed is","maxFileSize_infoMessage":"Max. file size","maxSelectionsError":"The maximum number of selections allowed is ","minCharactersError":"The number of characters should not be less than the minimum value:","minSelectionsError":"The minimum required number of selections is ","multipleError":"There are {count} errors in this page. Please correct them before moving on.","multipleFileUploads_emptyError":"{file} is empty, please select files again without it.","multipleFileUploads_fileLimitError":"Only {fileLimit} file uploads allowed.","multipleFileUploads_minSizeError":"{file} is too small, minimum file size is {minSizeLimit}.","multipleFileUploads_onLeave":"The files are being uploaded, if you leave now the upload will be cancelled.","multipleFileUploads_sizeError":"{file} is too large, maximum file size is {sizeLimit}.","multipleFileUploads_typeError":"{file} has invalid extension. Only {extensions} are allowed.","multipleFileUploads_uploadFailed":"File upload failed, please remove it and upload the file again.","nextButtonText":"Next","noSlotsAvailable":"No slots available","notEnoughStock":"Not enough stock for the current selection","notEnoughStock_remainedItems":"Not enough stock for the current selection ({count} items left)","noUploadExtensions":"File has no extension file type (e.g. .txt, .png, .jpeg)","numeric":"This field can only contain numeric values","oneError":"There is {count} error in this page. Please correct it before moving on.","pastDatesDisallowed":"Date must not be in the past.","pleaseWait":"Please wait...","prevButtonText":"Previous","progressMiddleText":"of","required":"This field is required.","requiredLegend":"All fields marked with * are required and must be filled.","requireEveryCell":"Every cell is required.","requireEveryRow":"Every row is required.","requireOne":"At least one field required.","restrictedDomain":"This domain is not allowed","reviewBackText":"Back to Form","reviewSubmitText":"Review and Submit","seeAllText":"See All","seeErrorsButton":"See Errors","selectionSoldOut":"Selection Sold Out","slotUnavailable":"{time} on {date} has been selected is unavailable. Please select another slot.","soldOut":"Sold Out","submissionLimit":"Sorry! Only one entry is allowed. &lt;br&gt; Multiple submissions are disabled for this form.","submitButtonText":"Submit","subProductItemsLeft":"({count} items left)","uploadExtensions":"You can only upload following files:","uploadFilesize":"File size cannot be bigger than:","uploadFilesizemin":"File size cannot be smaller than:","url":"This field can only contain a valid URL","validateEmail":"You need to validate this e-mail","wordLimitError":"Too many words. The limit is","wordMinLimitError":"Too few words.  The minimum is"});
      FormTranslation.init({"detectUserLanguage":"1","firstPageOnly":"1","options":"English (US)|English (UK)|Chinese|German (Germany)|Hindi|Swahili","originalLanguage":"en","primaryLanguage":"en","saveUserLanguage":"1","showStatus":"flag-with-nation","theme":"light-theme","version":"2"});
  /*INIT-END*/
  });

   JotForm.prepareCalculationsOnTheFly([null,null,{"name":"submit","qid":"2","text":"Send message","type":"control_button"},null,{"name":"name","qid":"4","subLabel":"","text":"Name","type":"control_textbox"},{"name":"email5","qid":"5","text":"E-mail","type":"control_email"},{"name":"clickTo6","qid":"6","text":"Your Information is Safe With us!","type":"control_text"},{"name":"getFree","qid":"7","text":"Send us a message","type":"control_head"},null,null,{"description":"","name":"typeA","qid":"10","subLabel":"","text":"Description","type":"control_textarea"},null,null,{"description":"","name":"typeA","qid":"13","subLabel":"","text":"Subject","type":"control_dropdown"},{"description":"","name":"typeA14","qid":"14","subLabel":"","text":"Description","type":"control_textarea"}]);
   setTimeout(function() {
JotForm.paymentExtrasOnTheFly([null,null,{"name":"submit","qid":"2","text":"Send message","type":"control_button"},null,{"name":"name","qid":"4","subLabel":"","text":"Name","type":"control_textbox"},{"name":"email5","qid":"5","text":"E-mail","type":"control_email"},{"name":"clickTo6","qid":"6","text":"Your Information is Safe With us!","type":"control_text"},{"name":"getFree","qid":"7","text":"Send us a message","type":"control_head"},null,null,{"description":"","name":"typeA","qid":"10","subLabel":"","text":"Description","type":"control_textarea"},null,null,{"description":"","name":"typeA","qid":"13","subLabel":"","text":"Subject","type":"control_dropdown"},{"description":"","name":"typeA14","qid":"14","subLabel":"","text":"Description","type":"control_textarea"}]);}, 20); 

    JotForm.showJotFormPowered = "new_footer";
  
    JotForm.poweredByText = "Powered by Jotform";
  
    JotForm.accessible = true;

    JotForm.ownerView=true;


    var all_spc = document.querySelectorAll("form[id='230183493383458'] .si" + "mple" + "_spc");
    for (var i = 0; i < all_spc.length; i++)
    {
      all_spc[i].value = "230183493383458-230183493383458";
    }